specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-01-10/examples/ReplicationProtectionContainerMappings_Update.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
The formatting in the doc needs to be fixed plus there's an extra '}' character causing a parsing failure, see the [log](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/355951732).